src/components/ui/rootHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+import React from "react";
+
+import { CircleUser } from "lucide-react";
+import { RoutePath } from "@/lib/route-path";
+import { getLoggedInSession } from "@/lib/auth";
+import Link from "next/link";
+
+const RootHeader = async () => {
+  const session = await getLoggedInSession();
+  const accountType = session?.accountType;
+
+  return (
+    <nav className="sticky top-0 z-10 bg-white">
+      <div className=" flex items-center justify-between border-b shadow-lg">
+        <Link href="/">
+          <Image
+            src="/bark-bank-logo.svg"
+            alt="bark bank logo"
+            width={60}
+            height={60}
+            className="ml-4 md:ml-10"
+          />
+        </Link>
+
+        <div className="hidden gap-x-4 sm:flex sm:last:mr-10">
```

Comment:
Could we change the gap to 8 (32px)?

Fig1: Screenshot from figma
<img width="668" alt="image" src="https://github.com/barkbank/barkbank-pawtal/assets/2757078/0b314c30-453a-43b7-b2f3-1efcec872d6b">

Reply:
Done [6c90f25](https://github.com/barkbank/barkbank-pawtal/pull/61/commits/6c90f25d287abf8373fb9862fd2eccaaf3a771fe)